tests/integration/src/test/java/org/apache/pulsar/tests/integration/containers/PulsarContainer.java

```diff
@@ -146,6 +146,26 @@ public void start() {
 
         beforeStart();
         super.start();
+        if (this.getContainerName().contains("presto-worker")) {
```

Comment:
I don't think this is the best way to get the logs of containers as they will be sprinkled across the whole test log.  We should just upload relevant log artifacts.  All the containers already dump its logs in compressed format (please reference: https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/utils/DockerUtils.java#L117) .  We just need to upload the logs as artifacts at end the end so that users can download all the logs to view/debug.  This is what was done when we were on Jenkins.

Reply:
Yes, that's right.